catroid/src/main/java/org/catrobat/catroid/io/StorageHandler.java

```diff
@@ -613,9 +613,8 @@ public static void copyDirectory(File destinationFile, File sourceFile) throws I
 	}
 
 	private void fixFolderStructureForSupportProject(String projectName, String sceneName) throws IOException {
-		projectName = UtilFile.encodeSpecialCharsForFileSystem(projectName);
-		sceneName = UtilFile.encodeSpecialCharsForFileSystem(sceneName);
-		String projectPath = buildProjectPath(projectName);
+		String projectNameEscaped = UtilFile.encodeSpecialCharsForFileSystem(projectName);
+		String projectPath = buildPath(Constants.DEFAULT_ROOT, projectNameEscaped);
```

Comment:
thanks for pointing this out, this makes the fix even smaller (only 2 deletions) ;)